subjects/evenlength/README.md

```diff
@@ -0,0 +1,35 @@
+## evenlength
+
+### Instructions
+
+Write a function that receives a slice of strings and returns a new slice with the strings in which the length is even.
+
+### Expected function
+
+```go
+func EvenLength(strings []string) []string {
+}
+```
+
+### Usage
+
+Here is a possible program to test your function:
+
+```go
+package main
+import (
+    "piscine"
+    "fmt"
+)
+func main() {
+    l := piscine.EvenLength([]string{"Hi", "Hola", "Ola", "Ciao", "Salut", "Hallo"})
```

Comment:
Remove the piscine imports and the variable, just print it

Reply:
Thank you.